iot/iot-device-registry-jdbc/src/main/java/io/enmasse/iot/registry/jdbc/devcon/impl/DeviceConnectionServiceImpl.java

```diff
@@ -65,7 +63,8 @@ private static JsonObject toGatewayResult(final String gatewayId) {
     }
 
     @Override
-    protected Future<DeviceConnectionResult> processSetCommandHandlingAdapterInstance(DeviceConnectionKey key, String adapterInstanceId, Span span) {
+    protected Future<DeviceConnectionResult> processSetCommandHandlingAdapterInstance(DeviceConnectionKey key, String adapterInstanceId, Duration lifespan, Span span) {
+        //TODO handle lifespan
```

Comment:
I think we should fix this before merging. And I think it would be relatively easy to do, by simply storing the TTL, and then return "none" when reading an expired TTL later on.